app/api/discount_codes.py

```diff
@@ -208,6 +210,14 @@ def before_get(self, args, kwargs):
             discount = db.session.query(DiscountCode).filter_by(code=kwargs.get('code'), deleted_at=None).first()
             if discount:
                 kwargs['id'] = discount.id
+                discount_tz = discount.valid_from.tzinfo
+                current_time = datetime.now().replace(tzinfo=discount_tz)
+                print(current_time)
```

Comment:
```suggestion
```

Reply:
@iamareebjamal My bad, Deleting it now, was using this during the testing.